### PR TITLE
PKGBUILD for Arch users

### PR DESCRIPTION
**Pull request by CleanMachine1**

### CODE REVIEW

**Comment by CleanMachine1** [comment]
Please ignore all of the extra commits, they were because I didn't pull before creating the branch. However I managed to fix it with merging the remote master branch,

**Review by Itai-Nelken — COMMENTED** [review]
We should add build (gcc, make, libc-dev) and runtime (libc) dependencies. I'm not sure how to do it, but I know it's possible.
Also I'm not sure about the package names, I think that libc might be libc6 and libc6-dev.

**Comment by CleanMachine1** [comment]
I'll have a look around, I have only just learnt how to use the install command for pkgbuild, before I was using sudo commands.

I think it will be pretty easy, I'll have a look around. I have been analyzing lots of random AUR pkgbuilds

File: PKGBUILD [hunk 1]
```diff
@@ -6,7 +6,7 @@ pkgdesc="Skip the dollar!"
 arch=('any')
 url="https://github.com/CleanMachine1/DollarSkip"
 license=('MIT')
-
+depends('make')
```
Comment (Itai-Nelken) [inline PKGBUILD:9]:
```suggestion
depends=('glibc')
makedepends=('gcc' 'make' 'glibc')
```

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,9 @@
...
-.vscode
\ No newline at end of file
+.vscode
+
+# PKGBUILD files
+
+src/
+pkg/
+3.0.tar.gz
+dollarskip-3.0-1-any.pkg.tar.zst
```
Comment (Itai-Nelken) [inline .gitignore:9]:
```suggestion
dollarskip*any.pkg.tar.zst
```

**Review by Itai-Nelken — APPROVED** [review]
tested and works.
maybe add a badge of the build status to the readme? here is one generated by github:
[Build](https://github.com/CleanMachine1/DollarSkip/actions/workflows/build.yml)
`[Build](https://github.com/CleanMachine1/DollarSkip/actions/workflows/build.yml)`

**Pull request merged**